.ci/bump-go-release-version.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# This script is in charge to read the latest go version supported in apm-server and upgrade its references
+# The contract will be always:
+#  1) Gather the go version from the upstream
+#  2) Update the references
+#  3) Commit changes
+
+set -euo pipefail
+
```

Comment:
This script queries the apm-server therefore there is no need to use any argument